doc/how-to-contribute.md

```diff
@@ -0,0 +1,117 @@
+
+<div align=center><href="https://sm.ms/image/3dyQAKplTnR2tX6" target="_blank"><img src="https://s2.loli.net/2022/08/16/3dyQAKplTnR2tX6.png" width="40%"></div>
+
+Finally, you can check it on github and make a pull request. Press "Compare & pull request" to make a PR.
+
+(Note: please commit pr to the devel branch)    
+
+<div align=center><href="https://sm.ms/image/Uj9m6zGtXRh1L3a" target="_blank"><img src="https://s2.loli.net/2022/08/16/Uj9m6zGtXRh1L3a.png" width="80%"></div>
+
+## How to contribute to DP-GEN tutorials and documents
+The documents of DP-GEN https://github.com/deepmodeling/dpgen/tree/master/doc
+- If you want to add the documentation of a toy model, simply put your file in the directory doc/toymodels/ and push;
+- If you want to add a new directory for a new category of instructions, make a new directory and add it in doc/index.rst.
+
+Tutorials repository: https://github.com/deepmodeling/tutorials
+The structure of tutorials and the preparation before writing a document see https://tutorials.deepmodeling.com/en/devel/Resources/writingTips.html#
+
+The latest page of DP-GEN Docs
+
+<div align=center><href="https://sm.ms/image/zEPKuj3TdaHI57b" target="_blank"><img src="https://s2.loli.net/2022/08/16/zEPKuj3TdaHI57b.png" width="60%"></div>
+
+### Examples of contributions
+Example 1(a merged one): https://github.com/deepmodeling/dpgen/pull/758
+Example 2(a simple one for beginner): https://github.com/deepmodeling/dpgen/pull/844
+### 1. Push your doc
+
+<div align=center><href="https://sm.ms/image/T4Zb8uiDXGeyYvc" target="_blank"><img src="https://s2.loli.net/2022/08/16/T4Zb8uiDXGeyYvc.png" width="60%"></div>
+
+### 2. Add the directory in index.rst
+
+<div align=center><href="https://sm.ms/image/q3iKvzQ8oRmfVLt" target="_blank"><img src="https://s2.loli.net/2022/08/16/q3iKvzQ8oRmfVLt.png" width="60%"></div>
+
+### 3. Build and check it
+
+As mentioned in "How to build the website to check if the modification works".
+
+### 4. Make pull request to dpgen
+
+https://github.com/deepmodeling/dpgen/pulls
+
+### How to build the website to check if it works
+
+1. Fork https://github.com/deepmodeling/dpgen
+2. Create account on readthedocs
+readthedocs.org
+https://readthedocs.org/
+3. Import https://github.com/<your-github-username>/dpgen.git 
+Remember to set Project homepage as https://github.com/<your-github-username>/dpgen , and set the Default branch as the one you do modification in Advanced Settings.
+
+<div align=center><href="https://sm.ms/image/4cVRb7ytT1h5l9n" target="_blank"><img src="https://s2.loli.net/2022/08/16/4cVRb7ytT1h5l9n.png" width="60%"></div>
+
+(“doc_overview” is just an example, please change it into your branch's name.)
+
+4. Build Version 
+
+<div align=center><href="https://sm.ms/image/CsJ7S5xeTYh9EWp" target="_blank"><img src="https://s2.loli.net/2022/08/16/CsJ7S5xeTYh9EWp.png" width="60%"></div>
```

Comment:
It's not necessary as we have PR build:
![image](https://user-images.githubusercontent.com/9496702/184803179-019a1172-65c3-429c-9faf-fbba0148da04.png)